.github/workflows/kurtosis-op-remote.yml

```diff
@@ -3,16 +3,11 @@
 name: Run kurtosis (remote images)
 
 on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
we don't have a main branch, and we don't want to run this on pr, only manual dispatch. I even was thinking to disable ci completely and only run on manual dispatch, sometimes I cancel workflows, but better idea is to just not open pr, test the branch with kurtosis with manual dispatch and then open pr when it's ready so ci runs only once hopefully.

Reply:
fixed!